rust/crates/report/src/report.rs

```diff
@@ -347,6 +347,26 @@ mod tests {
         report_data
     }
 
+    pub fn generate_mock_report_data_v13() -> ReportDataV10 {
```

Comment:
This should return `ReportDataV13` struct (which needs to be defined above), not `ReportDataV10`

```suggestion
    pub fn generate_mock_report_data_v13() -> ReportDataV13 {
```